mabwiser/utils.py

```diff
@@ -60,3 +62,148 @@ def reset(dictionary: Dict, value) -> NoReturn:
     """
     dictionary.update({}.fromkeys(dictionary, value))
 
+
+class _BaseRNG(metaclass=abc.ABCMeta):
```

Comment:
This is missing a method to get the state of the generator. See get_state method used in _Clusters class.

Reply:
can you point where it is needed/used (I wonder why no tests complained..)

Reply:
See clusters.py file line 30 and 32

Reply:
It was used in the Clusters class for copying the kmeans objects. You changed it to using the seed directly. However it's a standard function  that we may have future uses for

Reply:
Previously, this used "rng.get_state()[1][0] " which is an extremely hacky way to just get the seed. 

Currently, we pull that as rng.seed which does the job (that's why no test failed) and seems clean to me. 

What is the suggestion exactly?

Reply:
(Notice, we didn't need get_state() in any place, just the seed value..)

Reply:
We don't currently explicitly need it (agreed that the previous usage was hacky), but it's a common API function for random generators so it would be a good idea to have the function available

Reply:
I completely agree and we will most likely that functionality very soon. I just don't want to add a functionality that is not going to be used now --unless you deem it as a "blocker" for this PR.

Reply:
As long as we can directly access the seed from the random generator we don't currently have a need to get the state as I suggested.